datalab/mlalpha/_cloud_models.py

```diff
@@ -95,11 +89,36 @@ def delete(self, model_name):
       full_name = ('projects/%s/models/%s' % (self._project_id, model_name))
     return self._api.projects().models().delete(name=full_name).execute()
 
+  def list(self, count=10):
+    """List models under the current project in a table view.
+
+    Args:
+      count: upper limit of the number of models to list.
+    Raises:
+      Exception if it is called in a non-IPython environment.
+    """
+    import IPython
+    data = [{'name': model['name'], 
+             'defaultVersion': model['defaultVersion']['name'].split('/')[-1]
+                 if 'defaultVersion' in model else None}
+            for _, model in zip(range(count), self)]
+    IPython.display.display(
+        datalab.utils.commands.render_dictionary(data, ['name', 'defaultVersion']))
+    
+  def describe(self, model_name):
+    """Print information of a specified model.
+
+    Args:
+      model_name: the name of the model to print details on.
+    """    
+    model_yaml = yaml.safe_dump(self.get(model_name), default_flow_style=False)
+    print model_yaml
+    
 
 class CloudModelVersions(object):
   """Represents a list of versions for a Cloud ML model."""
 
-  def __init__(self, model_name, project_id=None, credentials=None, api=None):
+  def __init__(self, model_name, project_id=None):
     """Initializes an instance of a CloudML model version list that is iteratable
         ("for version in CloudModelVersions()").
```

Comment:
I don't think CloudModelVersions is iterable, so the comment "for version in CloudModelVersions()" is wrong? Likewise for CloudModels.